src/qt/forms/createwalletdialog.ui

```diff
@@ -38,6 +38,9 @@
      <height>24</height>
     </rect>
    </property>
+   <property name="placeholderText">
+     <string>MyWallet</string>
```

Comment:
this is also a new string,... though probably acceptable. Could use "Wallet Name" though,... but we already have this as label for the textfield.

Reply:
Ah yes, one alternative could be to allow empty names.